freqtrade/tests/rpc/test_rpc.py

```diff
@@ -59,7 +59,7 @@ def test_rpc_trade_status(default_conf, ticker, mocker) -> None:
     result_message = [
         '*Trade ID:* `1`\n'
         '*Current Pair:* '
-        '[ETH/BTC](https://bittrex.com/Market/Index?MarketName=BTC-ETH)\n'
+        '[BTC_ETH](https://www.bittrex.com/Market/Index?MarketName=BTC-ETH)\n'
```

Comment:
I think this should be `BTC/ETH` - underline as separator should only be used for filenames (in my opinion, anyway). The test is not passing (which is expected ...) - but if this really is the output then it should be changed.

Reply:
Yeah, messed up while merging. It should not change from the base branch. Thanks!